GMPWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -199,8 +204,11 @@ def run():
     print("Exception when getting trees: %r " % ex)
     exit(1)
 
-  print("\n".join(generateGaudiSteering(tree)))
+  wf_path = os.path.splitext(args[1])[0] + '.py'
```

Comment:
I think explicitly requiring an outputfilename is better then implicitly assuming the output file and potentially overwriting an existing file

Reply:
I agree, I changed it to have to indicate an output file